OpenRA.Game/GameRules/SoundInfo.cs

```diff
@@ -54,7 +81,16 @@ public string GetNext()
 				liveclips.AddRange(clips);
 
 			if (liveclips.Count == 0)
-				return null;		/* avoid crashing if there's no clips at all */
+				return null;        /* avoid crashing if there's no clips at all */
```

Comment:
Style nit: While we are touching this, please change the comment to the correct format.
```
// Avoid crashing if there's no clips at all
if (liveclips.Count == 0)
	return null;
```